README.md

```diff
@@ -594,10 +593,9 @@ Object.assign(b, a); // == b
 ```
 
 </details>
-
+  
```

Comment:
No real issue with those empty lines having a space or two added, but why not actually `trim()` every line so that this does not happen? This adds lots of unnecessary bytes in the file.

Reply:
yeah right I did this in the npm package because two empty spaces triggers a soft line break in Markdown

Reply:
but in the builder we can trim it

Reply:
Well, I think that's the last change we need before merging, otherwise we are exactly where we were before and with some cleaner code.